crates/node-core/src/metrics/prometheus_exporter.rs

```diff
@@ -104,6 +105,7 @@ where
     process.describe();
     describe_memory_stats();
     describe_io_stats();
+    record_version_metrics();
```

Comment:
nit: just to be consistent with `describe_gauge!` / `register_gauge!`

```suggestion
    register_version_metrics();
```